spec/models/course/level_spec.rb

```diff
@@ -6,7 +6,7 @@
 
   context 'before level_number is set' do
     it 'raises' do
-      expect { Course::Level.new.level_number }.to raise_error
+      expect { Course::Level.new.level_number }.to raise_error(RuntimeError)
```

Comment:
@allenwq @kxmbrian These errors in Ruby seem suitable:
 - ArgumentError
 - IndexError
 - RangeError